app/src/main/java/org/stepic/droid/model/code/ProgrammingLanguage.kt

```diff
@@ -106,13 +108,60 @@ enum class ProgrammingLanguage(val serverPrintableName: String) : Parcelable {
 
         }
 
-        private fun serverNameToLanguage(serverName: String): ProgrammingLanguage? {
-            return values()
-                    .find {
-                        it.serverPrintableName.equals(serverName, ignoreCase = true)
-                    } ?: return null
-        }
 
     }
 
 }
+
+private fun serverNameToLanguage(serverName: String): ProgrammingLanguage? {
+    return ProgrammingLanguage.values()
+            .find {
+                it.serverPrintableName.equals(serverName, ignoreCase = true)
+            } ?: return null
```

Comment:
Why is this additional `return null` with elvis operator here? If `find` returns `null` we may just return it.